runtime/gc_structs/ClassLoaderClassesIterator.cpp

```diff
@@ -33,15 +33,18 @@
 GC_ClassLoaderClassesIterator::GC_ClassLoaderClassesIterator(MM_GCExtensionsBase *extensions, J9ClassLoader *classLoader) :
 	_javaVM((J9JavaVM* )extensions->getOmrVM()->_language_vm)
 	,_classLoader(classLoader)
+	,_nextClass(firstClass())
```

Comment:
Please don't move `_nextClass`, we should not call `firstClass()` method here, as well as `_startingClass` initialization as well (because it depends at `_nextClass` initialization obviously).

Reply:
Updated.